packages/devtools-reps/src/object-inspector/utils.js

```diff
@@ -334,21 +492,30 @@ function getChildren({ getObjectProperties, actors, item }) {
   return children;
 }
 
+function getParent(item: ObjectInspectorItem) : ObjectInspectorItem | null {
+  return item.parent;
+}
+
 module.exports = {
```

Comment:
I can try to do that for the ObjectInspector folder. I don't know how well it will play with reps using requrie though, let's see :)